### PR TITLE
Heise.de:remove all exceptions

### PR DESCRIPTION
**Pull request by numismatika**
As it looks like the rewrites to plain http are gone all over the page, at least for now.
There are still traces of mixed-content left so we need to make sure that this is a) permanently enabled and b) neither ads nor mixed content breaks the page

### CODE REVIEW

**Comment by J0WI** [comment]
Sounds great! So far there is no official statement of them, as they did for `/security/`, but let's see.
Do not forgot to remove the `(partial)` flag and there are currently some failing tests.

**Comment by numismatika** [comment]
There are subdomains like tarifrechner or rl that are not covered yet, so the partial flag is still valid.
They probably are still experimenting as i have seen lets encrypt certificates on some domains.
I doubt they will make an aannouncement until mixed content issues are gone.

File: src/chrome/content/rules/Heise.de.xml [hunk 1]
```diff
@@ -78,23 +63,10 @@
...
 			<test url="http://www.heise.de/video/latest_vids.xml" />
 
-
-	<rule from="^http://m\.heise\.de/(js/|icons/|[ai]vw-bin/|security/icons/|stil/|fonts/)" to="https://m.heise.de/$1"/>
-		<exclusion pattern="^http://m\.heise\.de/$" />
-
-	<rule from="^http://(abo|piwik\.shop|prophet|shop)\.heise\.de/" to="https://$1.heise.de/"/>
-
-	<rule from="^http://www\.shop\.heise\.de/"
+	<rule from="^http://(www\.)?heiseshop\.de/"
```
Comment (J0WI) [inline src/chrome/content/rules/Heise.de.xml]: Please document this issue.

File: src/chrome/content/rules/Heise.de.xml [hunk 2]
```diff
@@ -1,46 +1,31 @@
...
-	<target host="piwik.shop.heise.de" />
+	<target host="stat.heiseshop.de" />
+	<target host="www.heise-gruppe.de" />
+	<target host="www.heise-medien.de" />
 	<target host="www.heise.de" />
-
-	<!--	Complications:
-				-->
+	<target host="www.heiseshop.de" />
 	<target host="www.shop.heise.de" />
```
Comment (J0WI) [inline src/chrome/content/rules/Heise.de.xml]: Please also group the targets by host, not only alphabetically.

**Comment by J0WI** [comment]
See also:
gutscheine.heise.de
jobs.heise.de (mismatch)

I think as long as the major part of the host works, it's safe to remove the partial flag.

File: src/chrome/content/rules/Heise.de.xml (hunk 2 above)
Comment (J0WI) [inline src/chrome/content/rules/Heise.de.xml]: @numismatika rule lgtm, could you solve this one before I merge?

Comment (numismatika) [inline src/chrome/content/rules/Heise.de.xml]: I reordered the target hosts.

**Comment by J0WI** [comment]
For ct there are also:
ho.ct.de
blog.ct.de
radio.ct.de
www.radio.ct.de
ctnotwin15.ct.de

**Pull request merged**